switchy/apps/bert.py

```diff
@@ -85,20 +89,34 @@ def on_park(self, sess):
         '''
         # assumption is that inbound calls will be parked immediately
         if sess.is_inbound():
-            sess.answer()  # next step will be in answer handler
+            if self.ring_response:
+                sess.broadcast(
+                    "{}::".format(self.ring_response), delay=self.pdd)
+
+            prd = self.prd
+            # next step will be in answer handler
+            if prd:
+                sess.broprdcast("answer::", delay=prd + (self.pdd or 0))
```

Comment:
omg... nice catch.
No definitely this is a typo.
Hehe guess it's pretty obvious I've changed that `prd` name too many times because this branch obviously isn't running.